source/admin_manual/list_of_events.rst

```diff
@@ -165,6 +165,126 @@ Global Fields
+
+
+auth_client_request_started
+---------------------------
+
+auth_client_request_challenged
+------------------------------
+
+auth_client_request_continued
+-----------------------------
+
+auth_client_request_finished
+----------------------------
+
++--------------------+-----------------------------------------------------------+
+| Field             | Description                                                |
++===================+============================================================+
+| user              | Username, if present                                       |
++-------------------+------------------------------------------------------------+
+| original_user     | Original username, if present                              |
++-------------------+------------------------------------------------------------+
+| auth_user         | Auth username, if present                                  |
++-------------------+------------------------------------------------------------+
+| error             | Error string if error occured                              |
++-------------------+------------------------------------------------------------+
+
+auth_client_userdb_list_started
+-------------------------------
+
++--------------+------------------------------------------------------------+
+| Field        | Description                                                |
++==============+============================================================+
+| user_mask    | User mask to list                                          |
++--------------+------------------------------------------------------------+
+
+auth_client_userdb_list_finished
+--------------------------------
+
++--------------+------------------------------------------------------------+
+| Field        | Description                                                |
++==============+============================================================+
+| user_mask    | User mask to list                                          |
++--------------+------------------------------------------------------------+
+| error        | Error string, if error occured                             |
++--------------+------------------------------------------------------------+
+
+auth_client_userdb_lookup_started
+---------------------------------
+
+auth_client_userdb_lookup_finished
+----------------------------------
+
++--------------------+-----------------------------------------------------------+
+| Field             | Description                                                |
++===================+============================================================+
+| user              | User to lookup                                             |
++-------------------+------------------------------------------------------------+
+| error             | Error string if error occured                              |
```

Comment:
Would be nice to try to standardize these "error" descriptions somehow. Now they're completely different everywhere. Although even within this pull request would be nice if it was the same :) Sometimes there's comma, sometimes not.

Reply:
@sirainen added lots of dots.

Reply:
There was also the comma issue :) Sometimes it's `Error string if error occured.` sometimes it's `Error string, if error occured.` - Consulting my nearby native english speaker it's apparently better without comma. Also, occurred with two r.

Reply:
grr, i'm totally blind to that =)